packages/website/src/components/magazine/index.js

```diff
@@ -1,28 +1,35 @@
-import React from 'react'
 import { getImage } from 'gatsby-plugin-image'
+import React from 'react'
+import { Heading } from '../../styles/sharedStyles'
 import ProjectCard from '../projects/components/ProjectCard/ProjectCard'
-import HeroSectionSlice from './hero-section-slice/HeroSectionSlice'
 import { CardWrapper, Wrapper } from './Elements'
-import { Heading } from '../../styles/sharedStyles'
+import HeroSectionSlice from './hero-section-slice/HeroSectionSlice'
 
 export default function MagazinePage ({ data }) {
   const date = data?.date
   const mainImage = getImage(data?.main_image)
   const title = data?.title?.text
   const info = data?.info?.text
   const link = data?.link?.url
+  const listTitle = data?.list_title?.text
+  const buttonName = data?.button_name?.text
+  const buttonTarget = data?.button_target ?? '_self'
+  const showButton = data?.show_button ?? false
 
   return (
     <>
       <HeroSectionSlice
         date={date}
-        mainImage={mainImage}
-        title={title}
-        info={info}
         link={link}
+        info={info}
+        title={title}
+        mainImage={mainImage}
+        buttonName={buttonName}
+        showButton={showButton}
+        buttonTarget={buttonTarget}
       />
       <Wrapper>
-        <Heading>Previous Magazines</Heading>
+        <Heading>{listTitle ?? ''}</Heading>
```

Comment:
Hey @nparashar150 , We can make our code cleaner by using ` ?? (nullish coalescing) ` in  ` listTitle ??  ' ' ` at the top like in the `showButton `.

Reply:
Sure thing, done ✅